qb/config.libs.sh

```diff
@@ -348,6 +348,11 @@ if [ "$HAVE_OPENGL" != 'no' ] && [ "$HAVE_OPENGLES" != 'yes' ]; then
    fi
 fi
 
+if [ "$HAVE_OPENGL" = 'no' ] && [ "$HAVE_OPENGLES3" = 'no' ]; then
```

Comment:
Somewhere in `qb/config.libs.sh` it would be good to add this line to prevent a build failure when a c++ compiler is not found.
```
check_enabled CXX OPENGL_CORE 'OpenGL core' 'The C++ compiler is' false
```
This can be easily reproduced with `CXX=false ./configure` and with this line will print this warning.
```
Notice: The C++ compiler is disabled, OpenGL core support will also be disabled.
```
Or this error if `--enable-opengl_core` is explicitly used.
```
Error: The C++ compiler is disabled and forced to build with OpenGL core support.
```